experimental/packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -351,30 +390,58 @@ export const getOutgoingRequestAttributes = (
     hostname: string;
     port: string | number;
     hookAttributes?: SpanAttributes;
-  }
+  },
+  semconvStability: SemconvStability
 ): SpanAttributes => {
   const hostname = options.hostname;
   const port = options.port;
   const requestMethod = requestOptions.method;
   const method = requestMethod ? requestMethod.toUpperCase() : 'GET';
   const headers = requestOptions.headers || {};
   const userAgent = headers['user-agent'];
-  const attributes: SpanAttributes = {
-    [SEMATTRS_HTTP_URL]: getAbsoluteUrl(
-      requestOptions,
-      headers,
-      `${options.component}:`
-    ),
+  const urlFull = getAbsoluteUrl(
+    requestOptions,
+    headers,
+    `${options.component}:`
+  );
+  const oldAttributes: SpanAttributes = {
+    [SEMATTRS_HTTP_URL]: urlFull,
     [SEMATTRS_HTTP_METHOD]: method,
     [SEMATTRS_HTTP_TARGET]: requestOptions.path || '/',
     [SEMATTRS_NET_PEER_NAME]: hostname,
     [SEMATTRS_HTTP_HOST]: headers.host ?? `${hostname}:${port}`,
   };
 
+  const newAttributes: Attributes = {
+    // Required attributes
+    [ATTR_HTTP_REQUEST_METHOD]: method,
+    [ATTR_SERVER_ADDRESS]: hostname,
+    [ATTR_SERVER_PORT]: Number(port),
+    [ATTR_URL_FULL]: urlFull,
+    // leaving out protocol version, it is not yet negotiated
+    // leaving out protocol name, it is only required when protocol version is set
+    // retries and redirects not supported
+
+    // Opt-in attributes left off for now
+  };
```

Comment:
Quick note that `url.path` is missing here at the moment but required

Reply:
`url.path` is not listed in the required attributes according to https://github.com/open-telemetry/semantic-conventions/blob/main/docs/http/http-spans.md#http-client

Reply:
My bad